hachoir/stream/output.py

```diff
@@ -112,14 +112,11 @@ def writeInteger(self, value, signed, size_byte, endian):
         self.writeBytes(raw)
 
     def copyBitsFrom(self, input, address, nb_bits, endian):
-        if (nb_bits % 8) == 0:
-            self.copyBytesFrom(input, address, nb_bits // 8)
```

Comment:
This code is an optimization, why removing it?

Reply:
Currently, it is unable to handle all cases. Specifically, it crashes when output._bit_pos is not byte-aligned. I'll have another look to see if I can disable it only when needed.

Reply:
Okay, I tried that and it simply worked... not sure why I didn't include it before. Commit upcoming!

Reply:
Fixed in https://github.com/vstinner/hachoir/pull/54/commits/720a5905d928e82d48b9a387a1a3c80881f504be